### PR TITLE
Update tutorial 01 and 02

### PR DESCRIPTION
**Pull request by Persdre**

### CODE REVIEW

**Comment by Persdre** [comment]
Ok! I am looking at it! @asherpasha

File: 01-tutorial.ipynb [hunk 1]
```diff
@@ -11,14 +11,14 @@
...
-    "Welcome to your first intermine-python tutorial. Over a series of approximately 12 tutorials, we will go through the basics of writing code in Python that allows us to query the intermine database. "
+    "Welcome to the first intermine-python tutorial. Over a series of approximately 12 tutorials, we will go through the basics of writing code in Python that allows us to query the intermine database. "
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This tutorial will tell you about the basics of intermine-python queries and how to write your first query. To get started, you would want to \"pip install\" the intermine package. Once you have installed the package, you are good to go! "
+    "This tutorial will tell you about the basics of intermine-python queries and how to write your first query. To get started, you would want to \"pip install intermine\" in terminal first. Once you have installed the package, you are good to go! "
```
Comment (yochannah) [inline 01-tutorial.ipynb:21]:
```suggestion
    "This tutorial will tell you about the basics of intermine-python queries and how to write your first query. To get started, you would want to `pip install intermine` in terminal first. Once you have installed the package, you are good to go! "
```

Comment (yochannah) [reply 01-tutorial.ipynb:21]: small suggestion - maybe make this markdown :)

**Comment by asherpasha** [comment]
Thanks. It worked.

**Pull request merged**